calculator-app/src/Buttonnumbers.js

```diff
@@ -0,0 +1,66 @@
+import './App.css';  // import App.css to get the class
+import { useContext  } from "react";   // import useContext 
+import { ButtonContext } from "./App";    // import ButtonContext which was created in the App.js
+
+function Buttonnumbers() {
+  const buttoncont = useContext(ButtonContext);   // Define useContext with imported context (ButtonContext)
+  const {digit, setDigit}=buttoncont      // destructuring the (buttoncont) context variable which are needed (values need to be changed)
+  const numbers = ['/','*','-','+',9,8,7,4,5,6,1,2,3,0,'%','.','='];    // to create array element
+  const operators = ['/','*','-','+','%','.','='];
```

Comment:
use separate constants file for constant variables.
Step1: Create **utility** Folder.
Step2: Create **constants.js**
step3:  **export** const numbers = ['/','*','-','+',9,8,7,4,5,6,1,2,3,0,'%','.','='];   
Step 4: Wherever need : import { numbers } from './constants';
and start using the constant variable.